.gitattributes

```diff
@@ -0,0 +1 @@
+NOTICE.html linguist-generated=true
```

Comment:
Oh yes, it's here: https://github.com/github/linguist/blob/master/docs/overrides.md#summary

`linguist-generated` means it will _not_ show up in diffs/PRs, which wasn't what I wanted. `linguist-vendored` does still show up in PRs but is excluded from repo stats.

Reply:
https://github.com/github/linguist/blob/master/docs/overrides.md#generated-code

Files marked as `linguist-generated` are suppressed in diffs, files marked as `linguist-vendored` are not.

Reply:
Yeah, I don't generally look at the diff, but I still want to see it in my PRs when it's being changed.